packages/z/zellij/files/dev.zellij.Zellij.metainfo.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
```

Comment:
zellij is a terminal application; is this the right component?

Reply:
Oh, I thought it was more like `kitty`. Now that i tried it, it is more like `helix`. Yeah, this is not the right component. But then again, it provides desktop file which console application are not required to. For console application, [this](https://www.freedesktop.org/software/appstream/metainfocreator/#/consoleapp) the template for it.

Reply:
I think the component can be changed to `<component type="console-application">` and will still be valid. Let me try in a sec.